plugins/woocommerce/includes/class-wc-emails.php

```diff
@@ -268,9 +268,10 @@ public function get_from_address() {
 	 * Get the email header.
 	 *
 	 * @param mixed $email_heading Heading for the email.
+  	 * @param mixed $email         Email object.
 	 */
-	public function email_header( $email_heading ) {
-		wc_get_template( 'emails/email-header.php', array( 'email_heading' => $email_heading ) );
+	public function email_header( $email_heading, $email ) {
```

Comment:
```suggestion
	public function email_header( $email_heading, $email = null ) {
```

Making the new parameter optional should prevent any compatibility issues with custom templates.